Client/Info.plist

```diff
@@ -46,12 +34,16 @@
 	<key>CFBundleURLTypes</key>
 	<array>
 		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
 			<key>CFBundleURLName</key>
 			<string>org.mozilla.Client</string>
 			<key>CFBundleURLSchemes</key>
 			<array>
 				<string>firefox</string>
 				<string>$(MOZ_INTERNAL_URL_SCHEME)</string>
+				<string>http:</string>
+				<string>https:</string>
```

Comment:
Relevant changes can be found here

Reply:
@kaylagalway Would it be possible to add these without changing full structure of Info.plist?

Reply:
Hmm good question. Let me try to backtrack and manually add them and see if it causes any issues.

Reply:
Okay - this change is made. I'm having separate xcode 12 beta build issues - so can you let me know if this causes any build problems for you for any reason?